web/tests/functional/diff_local_remote/test_diff_local_remote.py

```diff
@@ -574,6 +574,38 @@ def test_diff_codeclimate_output(self):
 
         shutil.rmtree(export_dir)
 
+    def test_diff_no_trim_codeclimate_output(self):
+        """ Test codeclimate output when using diff and don't set env vars. """
+        base_run_name = self._run_names[0]
+
+        export_dir = os.path.join(self._local_reports, "export_dir")
+
+        diff_cmd = [self._codechecker_cmd, "cmd", "diff",
+                    "--unresolved",
+                    "--url", self._url,
+                    "-b", base_run_name,
+                    "-n", self._local_reports,
+                    "-o", "codeclimate",
+                    "-e", export_dir]
+
+        self.run_cmd(diff_cmd, self._env)
+        codeclimate_issues_file = os.path.join(export_dir,
+                                               'codeclimate_issues.json')
+        self.assertTrue(os.path.exists(codeclimate_issues_file))
+
+        with open(codeclimate_issues_file, 'r',
+                  encoding="utf-8", errors="ignore") as rw_file:
```

Comment:
File was opened  only 'r' flag. issues_file would be a better name.

Reply:
Most of the case `f` is used to represent a file object. So maybe we can follow this convention too.